hack/build-rpm-release.sh

```diff
@@ -24,7 +31,7 @@ tito tag --use-version="${OS_RPM_VERSION}" \
 tito_tmp_dir="${BASETMPDIR}/tito"
 mkdir -p "${tito_tmp_dir}"
 tito build --output="${tito_tmp_dir}" --rpm --no-cleanup --quiet --offline \
-           --rpmbuild-options="--define 'make_redistributable ${make_redistributable}'"
+           --rpmbuild-options="${RPM_BUILD_OPTS}"
```

Comment:
I think we might be able to get rid of the `if` statement and just use:

```sh
tito build --output="${tito_tmp_dir}" --rpm --no-cleanup --quiet --offline \
           --rpmbuild-options="--define 'make_redistributable ${make_redistributable}'  ${RPM_BUILD_OPTS:-}"
```

As long as the trailing space in the empty $RPM_BUILD_OPTS` expansion case isn't a problem for `rpmbuild`.